src/mpi/errhan/errnames.txt

```diff
@@ -378,6 +378,8 @@ be in the range 0 to %d
 **infonokey %s:MPI_Info key %s is not defined 
 **infonkey:Requested nth key does not exist
 **infonkey %d %d:Requested key %d but this MPI_Info only has %d keys
+**infonoteq:MPI_Info are not consistent for all processes
+**infonoteq %s:MPI_Info key %s is not consistent for all processes
```

Comment:
A minor comment here: maybe we want to keep the `infonoteq` consistent and more specific. E.g.,
```
**infonoteq: The MPI_Info key value are not consistent for all processes
**infonoteq %s: The MPI_Info value with key %s is not consistent for all processes
```

Reply:
Updated as you suggested.